src/include/capacity.h

```diff
@@ -53,6 +55,8 @@ struct __wt_capacity {
     uint64_t total;      /* Bytes/sec total capacity */
     uint64_t threshold;  /* Capacity size period */
 
+    uint64_t fsync_background_period; /* when exceed the period time(sec), force to fsync */
```

Comment:
This comment could use some improvements - we can get away with something shorter since the full description is in the config. The below is a suggestion, but feel free to do your own thing.

`Maximum period between fsync calls`